TruckersFM/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "author": {
+    "name": "Ninja",
+    "id": "150987570715688960"
+  },
+  "service": "TruckersFM",
+  "description": {
+    "en": "TruckersFM is your staiton for driving."
```

Comment:
saiton - > station

Reply:
Maybe even their official description?
"The number one hit music station for Trucking Simulators. Radio made for driving, now playing online, in cab and on the go."